src/wolfgame.py

```diff
@@ -2161,6 +2162,9 @@ def show_votes(cli, nick, chan, rest):
         the_message = ", ".join(votelist)
         if len(pl) >= var.MIN_PLAYERS:
             the_message += "{0}Votes needed for a majority: {1}".format("; " if votelist else "", int(math.ceil(len(pl)/2)))
+
+        if len(var.START_VOTES) > 0:
+            the_message += "; Votes to start the game: {} ({})".format(len(var.START_VOTES), ', '.join(var.START_VOTES))
```

Comment:
That can't really happen as it always at least prints "Votes needed for a majority: x" before (except for when there's not enough players, but then it also doesn't print the start votes).